core/src/main/java/pl/agh/edu/ui/screen/init/GameCreationWizard.java

```diff
@@ -0,0 +1,44 @@
+package pl.agh.edu.ui.screen.init;
+
+import java.util.Optional;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+
+import pl.agh.edu.GdxGame;
+import pl.agh.edu.engine.hotel.HotelType;
+import pl.agh.edu.engine.hotel.dificulty.DifficultyLevel;
+import pl.agh.edu.ui.panel.DifficultyPanel;
+import pl.agh.edu.ui.panel.ScenarioPanel;
+import pl.agh.edu.ui.screen.main.MainScreen;
+
+public class GameCreationWizard {
```

Comment:
kiedy mówiłem o wizardzie bardziej chodziło mi o pseudo krokowiec w którym swobonie możemy przechodzić miedzy stanami, bez potrzeby zmiany przycisków ("Next","Previous") i tego title a jedynie zawartości w środku i zbieraniu poszczególnych parametrów z tego krokowca, ale tak też jest spoczko tylko zmieniłbym nazwe bo to już nie jest wizard a jakiś kontener